scanner/src/main/kotlin/scanners/Askalono.kt

```diff
@@ -154,7 +154,7 @@ class Askalono(name: String, config: ScannerConfiguration) : LocalScanner(name,
         return yamlMapper.createArrayNode().apply { addAll(yamlNodes) }
     }
 
-    override fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
+    private fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
```

Comment:
Commit message: Change "only called class internally by each scanner implementations" to "only called internally by each scanner implementation".

Reply:
done.